### ISSUE TITLE
Status is not correctly updated after update to 6.6.0

### ISSUE THREAD

**Issue by jfrede** [RocketChat/Rocket.Chat#31705]
### Description:

After updating to version 6.6.0 the status of a person is only updated when the page is reloaded. 
In version 5.5.x this worked fine. 

### Steps to reproduce:

1. update to Version 6.6.0
2. update your status
3. watch status on another user

### Expected behavior:

Status update when somebody changes status

### Actual behavior:

Status will not be updated, only when i reload the page. 

### Server Setup Information:

- Version of Rocket.Chat Server:  6.6.0
- Operating System:  debian 12
- Deployment Method: docker
- Number of Running Instances: 1 
- DB Replicaset Oplog:  128
- NodeJS Version: from docker 6.6.0 Container
- MongoDB Version: 5.0 (docker)

### Client Setup Information

- Desktop App or Browser Version:  Google Chrome  Version 121.0.6167.139 (Offizieller Build) (64-Bit)
- Operating System:  Linux (OpenSUSE Leap 15.5 and Debian12)

**Comment by lovekyleman** [comment]
Users always online here

**Comment by Gummikavalier** [comment]
I confirm this issue too in RC 6.6.0. Reload makes it change, but otherwise remains stuck to current state.

**Comment by k3tiv** [comment]
Same problem here
server: 6.6.0
desktop app: 3.9.12

![image](https://github.com/RocketChat/Rocket.Chat/assets/36487398/031a4358-d2a9-4f08-b563-f81176b52d6e)

**Comment by PranitPatil03** [comment]
Hello @jfrede 

I have tested this in my own environment and noticed that the User status updates in real-time. I also have a video demonstrating the entire process.

https://github.com/RocketChat/Rocket.Chat/assets/91155068/bbd68b3c-0014-4d23-9538-7aaba260362b

**Comment by JoeLynnTurner1** [comment]
I have the same problem after upgrading to version 6.6.0. Status are not updated, only need reload the page.

**Comment by jfrede** [comment]
I've also done a Video with the Server log open. 

https://github.com/RocketChat/Rocket.Chat/assets/1298727/123aaab5-adf6-4500-86a5-1cade05e53f2

**Comment by chotaire** [comment]
We're suffering the same problem, upgraded to 6.6.0 via docker-compose.

**Comment by xcpdq** [comment]
Having the same issue too. After upgrading to 6.0.0 (Enterprise license) via docker-compose almost all of our team members are showing as online. Tested Reload and Reload ignoring cache, same issue. Also, notifications don't work well anymore too. DM's no longer have the sound.

**Comment by urosino** [comment]
> Having the same issue too. After upgrading to 6.0.0 (Enterprise license) via docker-compose almost all of our team members are showing as online. Tested Reload and Reload ignoring cache, same issue. Also, notifications don't work well anymore too. DM's no longer have the sound.

Same here. I hope this will be resolved soon.

**Comment by mvvvmd** [comment]
Same issue with community server v6.6.0, snap on Ubuntu 20.04, with the latest clients & in the browser. We tested the client snap, flatpak & .deb, multiple users affected. We also reset user status because this resolved status issues before.

```
sudo rocketchat-server.mongo

use parties

db.usersSessions.deleteMany({})
db.users.updateMany({status:"online"},{$set:{status:"offline"}})
exit
```

**Comment by sampaiodiego** [comment]
thx guys for letting me know about this.. I noticed this as well last week and asked some colleagues to take a look, they came up with the following PR the same day this issue was opened 😬  https://github.com/RocketChat/Rocket.Chat/pull/31723

we'll probably have a patch release soon

**Comment by Gummikavalier** [comment]
Tested working. Shift-reload was required for the web client. Maybe plain F5 would have been enough too.

**Comment by lovekyleman** [comment]
> Tested and working in RC 6.6.1.
> 
> 
> 
> Shift-reload was required for the web client. Maybe plain F5 would have been enough too.

6.6.1. On rocketchat iOS app I see all users online. Users status bug isn't fixed

**Comment by sampaiodiego** [comment]
@jfrede thanks for detailed report and video.. I'd appreciate if you can test out 6.6.1, the issue you showed was fixed on that version.

everyone else still having issues pls add more details on issue https://github.com/RocketChat/Rocket.Chat/issues/31768

**Comment by jfrede** [comment]
Sry i can't test this on my environment because the docker containers are only build for releases not for release candidates and i don't want to build the container by my own.

**Comment by sampaiodiego** [comment]
@jfrede 6.6.1 is not actually a release candidate.. 6.6.1 final is already available as Docker containers

**Comment by jfrede** [comment]
Yes installed it today and works fine tnx for your work.

**Comment by pythys** [comment]
The problem is not resolved in 6.6.1

**Comment by chotaire** [comment]
Yeah, it has returned to us too, using 6.6.1. People always showing as online and active, if they were ever online (probably using the web browser). Even in the middle of the night.

**Comment by sampaiodiego** [comment]
thx for the reports everyone, this is the fix for this new buggy behavior https://github.com/RocketChat/Rocket.Chat/pull/31833
